src/test/components/__snapshots__/FlameGraph.test.js.snap

```diff
@@ -69,7 +69,7 @@ exports[`renders FlameGraph correctly 1`] = `
     <div
       className="flameGraphLabels grippy"
       title="thread: \\"Empty\\" (0)
-    process: \\"default\\" (0)"
+process: \\"default\\" (0)"
```

Comment:
Looks like they changed how they generate snapshot for multiline string attributes